DedicatedServer/Scripts/Libs/Zrx/ModeLibs/Common/WarmUpReadyUp/WarmUpReadyUp_UI.Script.txt

```diff
@@ -1,18 +1,19 @@
+// #RequireContext CSmMlScriptIngame
+
 #Const C_ReadyEvent_Type	"ready"
 #Const C_ManialinkName		"WarmUpReadyUp"
 
 Text GetManialink() {
 	return """
 	<?xml version="1.0" encoding="utf-8" standalone="yes" ?>
 	<manialink version="3" name="{{{C_ManialinkName}}}">
-		<frame id="player-ready-frame" pos="0 -60">
+		<frame id="player-ready-frame" pos="0 65">
+			<label pos="0 -8" z-index="0" size="40 5" text="pad  or  + " halign="center" textsize="4" textfont="GameFontRegular" valign="center2"/>
```

Comment:
Please check how it looks when S_RoundsToWinMap > 0, because that would move the header a bit down since the Round wins would be displayed above it.

Also, I feel like it's too much. Please remove the size of the "F7" and the text below the button. Also we can probably ommit the words for the pad shortcuts to reduce the size: "<pad-select> / <pad-L1> + <pad-R1>"

Additionally, the part text below the button is weirdly vertically aligned; The pad buttons look alinged to the top while the text seems to be centers. Is this maybe because of `valign="center2"`? How would normal `center` look like?

![image](https://user-images.githubusercontent.com/53338174/147229050-03ada597-ae0a-49cc-9b58-79b06ed4fa06.png)

Reply:
It's actually the glyphs being off-center.